pandas/io/html.py

```diff
@@ -554,10 +600,9 @@ def _text_getter(self, obj):
         return obj.text_content()
 
     def _parse_td(self, row):
-        return row.xpath('.//td|.//th')
-
-    def _parse_tr(self, table):
-        return table.xpath('.//tr')
+        # Look for direct descendents only: the "row" element here may be a
```

Comment:
Are you looking for children or descendants? Conflict between comment and code here?

Reply:
That's the word I was looking for! d'oh, thanks :)